cmd/snake-runner/scheduler.go

```diff
@@ -104,6 +108,43 @@ func (snake *Snake) startScheduler() error {
 					shell.PREFERRED_SHELL,
 				)
 			}
+
+			container, err := execer.Create(ctx, executor.CreateOptions{})
+			if err != nil {
+				return karma.Format(
+					err,
+					"unable to create container for ssh-agent validation",
```

Comment:
this is not always a container

Reply:
I've changed 'container' to 'session', but in this particular case this error message will never appear because it shell `executor.Create` always returns 'nil'.

Reply:
I see, ok